fvtest/tril/tril/compiler_util.hpp

```diff
@@ -73,6 +74,26 @@ static std::vector<TR::DataTypes> parseArgTypes(const ASTNode* node) {
    return argTypes;
 }
 
+/**
+ * @brief  Convert a linkage convention name to a LinkageConvention 
+ *         appropriate for setting on a MethodSymbol. 
+ *
+ * @return the linkage convention, or TR_None if the name is not 
+ *         recognized. 
+ */
+static TR_LinkageConventions convertStringToLinkage(const char * linkageName) { 
+   std::string ln = linkageName;  
+   if (ln == "private") 
+      return TR_Private; 
+   if (ln == "system")
+      return TR_System;
+   if (ln == "allregister") 
+      return TR_AllRegister;
```

Comment:
Given we only have a test for "system" linkage, maybe it would be better to just define "system" linkage here and then add other linkages as we write tests that use and validate them?

Reply:
Done